### PR TITLE
Ad matching for ads that are not categorized

### PR DESCRIPTION
**Pull request by tmancey**
Fixes https://github.com/brave/brave-browser/issues/5183

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [x] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:

- Confirm targeted ads for **PARENT-CHILD** page classifications are served for the United States of America, Canada, United Kingdom, Germany, France, Australia, New Zealand and Ireland
- Confirm targeted ads for **PARENT** page classifications are served for the United States of America, Canada, United Kingdom, Germany, France, Australia, New Zealand and Ireland
- Confirm untargeted ads are served for the United States of America, Canada, United Kingdom, Germany, France, Australia, New Zealand and Ireland if no eligible ads are available
- Confirm untargeted ads are served for Argentina, Austria, Brazil, Switzerland, Chile, Colombia, Denmark, Ecuador, Israel, India, Italy, Korea, Mexico, Netherlands, Peru, Philippines, Poland, Sweden, Singapore, Venezuela and South Africa

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by tmancey** [comment]
@jsecretan @lukemulks The above issue is development complete, we need a list of untargeted regions for initial release. Thanks

File: vendor/bat-native-ads/README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ void ChangeLocale(
 
 `ClassifyPage` should be called when a page has loaded in the current browser tab, and the HTML is available for analysis
```
Comment (emerick) [inline vendor/bat-native-ads/README.md]: Function name in comment needs an update...

Comment (tmancey) [reply vendor/bat-native-ads/README.md]: Fixed

**Comment by tmancey** [comment]
Added support for Argentina, Austria, Brazil, Switzerland, Chile, Colombia, Denmark, Ecuador, Israel, India, Italy, Korea, Mexico, Netherlands, Peru, Philippines, Poland, Sweden, Singapore, Venezuela and South Africa to Ads (untargeted)

**Pull request merged**